src/components/Table/Table.tsx

```diff
@@ -61,40 +66,11 @@ const Table: React.FunctionComponent = () => {
       <StateContext.Provider value={state}>
         <Game />
         {isDeveloperMode && <DeveloperMode />}
-        <div
-          css={css`
-            background-color: var(--dark);
-            height: 37.5rem;
-            width: 50rem;
-            position: relative;
-          `}
-        >
+        <TableContainer>
           <Connections />
-          <div
-            css={css`
-              color: white;
-              position: absolute;
-              top: 0.25rem;
-              left: 0.25rem;
-              z-index: 4;
-              font-size: var(--font-size-xs);
-            `}
-          >
-            {gameType}
-          </div>
-          <div
-            css={css`
-              position: absolute;
-              width: 100%;
-              height: 100%;
-              z-index: 1;
-            `}
-          >
-            <div
-              css={css`
-                position: absolute;
-              `}
-            />
+          <GameTypeWrapper>{gameType}</GameTypeWrapper>
+          <TableWrapper>
+            <Absolute />
```

Comment:
I kept it for no reason :) I think it should go, unless it is some empty div magic I dont understand @norbertdragan :)